hack/deploy-hub-local/lab-nfs.sh

```diff
@@ -10,10 +10,17 @@ export PRIMARY_IP=192.168.150.1
 dnf -y install nfs-utils
 systemctl enable --now nfs-server
 export MODE="ReadWriteOnce"
+MAXACTIVE=5
```

Comment:
```suggestion
MAXACTIVEPV=5
```
lets give it a name that we can know what it means